ocaml/xcp-rrdd/scripts/rrdd/rrdd.py

```diff
@@ -75,28 +75,13 @@ def connect(self):
     self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
     self.sock.connect(path)
 
-class UDSHTTP(httplib.HTTP):
-  _connection_class = UDSHTTPConnection
-
 class UDSTransport(xmlrpclib.Transport):
-  def __init__(self, use_datetime = 0):
-    self._use_datetime = use_datetime
-    self._extra_headers = []
-
+  # FIXME: is this function actually used anywhere?
```

Comment:
I think `rrdd.py` is just an example library that one can use to query RRDs from XAPI., I don't think it is actually used by anything in the product, but I could be wrong, so lets keep the method here for now, we can always remove it later.